templates/hubspot_tracking_code.twig

```diff
@@ -0,0 +1,3 @@
+{% if hubspot_tracking_code and 'gravityforms' in p4_blocks %}
```

Comment:
No need to check for `gravityforms` block. The snippet should be included everywhere (see also the comments on the ticket).